### PR TITLE
Initial Epix assets and chain info.

### PR DESCRIPTION
**Pull request by MudDev**

### CODE REVIEW

File: testnets/epixtestnet/assetlist.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      "name": "Epix Testnet",
+      "display": "tepix",
+      "symbol": "TEPIX",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/epixtestnet/images/epix.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/epixtestnet/images/epix.svg"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/epixtestnet/images/epix.png",
```
Comment (JeremyParish69) [inline testnets/epixtestnet/assetlist.json:27]: Please remove the padding from the image files, as that won't be clipped in most UIs

File: testnets/epixtestnet/chain.json [hunk 2]
```diff
@@ -0,0 +1,109 @@
+{
+  "$schema": "../../chain.schema.json",
+  "chain_name": "epixtestnet",
+  "status": "live",
+  "network_type": "testnet",
+  "website": "https://epix.zone/",
+  "pretty_name": "Epix Testnet",
+  "chain_type": "cosmos",
+  "chain_id": "epix_4243-0",
+  "bech32_prefix": "epix",
```
Comment (JeremyParish69) [inline testnets/epixtestnet/chain.json:10]: Please register this prefix: https://github.com/satoshilabs/slips/blob/master/slip-0173.md

File: testnets/epixtestnet/chain.json [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+      }
+    ]
+  },
+  "codebase": {
+    "git_repo": "https://github.com/EpixZone/Epix",
+    "recommended_version": "v12.0.0-rc4",
+    "compatible_versions": [
+      "v12.0.0-rc4"
+    ],
+    "cosmos_sdk_version": "0.46",
```
Comment (JeremyParish69) [inline testnets/epixtestnet/chain.json:43]: please add the cosmos_sdk, consensus, and ibc version data into the v1 object in the versions array.
Note that these legacy properties will also require the replacement properties (e.g, cosmos_sdk_version -> sdk: type, version; etc.)

File: testnets/epixtestnet/chain.json (hunk 2 above)
Comment (MudDev) [reply testnets/epixtestnet/chain.json:10]: Great call out. This is now complete: https://github.com/satoshilabs/slips/pull/1799

File: testnets/epixtestnet/assetlist.json (hunk 1 above)
Comment (MudDev) [reply testnets/epixtestnet/assetlist.json:27]: Uploaded new image.

File: testnets/epixtestnet/chain.json (hunk 3 above)
Comment (MudDev) [reply testnets/epixtestnet/chain.json]: Fixed

File: epix/assetlist.json [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/epix/images/epix.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/epix/images/epix.svg",
+          "theme": {
+            "primary_color_hex": "#69e9f5"
+          }
+        }
+      ],
+      "coingecko_id": "epix",
```
Comment (JeremyParish69) [inline epix/assetlist.json:34]: This cgid does not exist in https://api.coingecko.com/api/v3/coins/list. Please remove.

File: testnets/epixtestnet/assetlist.json [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+          "denom": "tepix",
+          "exponent": 18
+        }
+      ],
+      "base": "atepix",
+      "name": "Epix Testnet",
+      "display": "tepix",
+      "symbol": "TEPIX",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/epixtestnet/images/epix.png",
```
Comment (JeremyParish69) [inline testnets/epixtestnet/assetlist.json:22]: The images being uploaded to the epixtestnet/images/ directory are identical to the mainnet images, so we don't need the duplicates. Please use image_sync and point to the mainnet asset, and update the path to the mainnet images. E.g.,
```
  "image_sync": {
    "chain_name": "epix",
    "base_denom": "aepix"
  },
  "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/epix/images/epix.png",
  "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/epix/images/epix.svg"
```

Comment (JeremyParish69) [reply testnets/epixtestnet/assetlist.json:22]: Please remove the testnets/epixtestnet/images/ directory one all references to it have been updated.

Comment (MudDev) [reply testnets/epixtestnet/assetlist.json]: Thanks for the suggestions, completed these changes.

File: epix/chain.json [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+    "versions": [
+      {
+        "name": "v1.0.0",
+        "recommended_version": "v1.0.0",
+        "compatible_versions": [
+          "v1.0.0"
+        ],
+        "next_version_name": "",
+        "sdk": {
+          "type": "cosmos_sdk",
```
Comment (JeremyParish69) [inline epix/chain.json:56]: type is just "cosmos".
as per https://github.com/cosmos/chain-registry/blob/master/chain.schema.json#L655

File: epix/chain.json [hunk 7]
```diff
@@ -0,0 +1,119 @@
...
+          "version": "0.46"
+        },
+        "consensus": {
+          "type": "tendermint",
+          "version": "0.34"
+        },
+        "ibc": {
+          "version": "v6.1.0"
```
Comment (JeremyParish69) [inline epix/chain.json:64]: "type" is also required. Most common is "go"

File: epix/chain.json (hunk 6 above)
Comment (MudDev) [reply epix/chain.json]: Fixed, thank you.

File: epix/chain.json (hunk 7 above)
Comment (MudDev) [reply epix/chain.json:65]: Fixed, thank you.

File: epix/assetlist.json [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+        {
+          "denom": "aepix",
+          "exponent": 0
+        },
+        {
+          "denom": "epix",
+          "exponent": 18
+        }
+      ],
+      "base": "aepix",
```
Comment (JeremyParish69) [inline epix/assetlist.json:17]: both this and the testnet asset require that type_asset be defined.

Comment (MudDev) [reply epix/assetlist.json:17]: Added type_asset, thanks!

File: testnets/epixtestnet/assetlist.json [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+          "exponent": 0
+        },
+        {
+          "denom": "epix",
+          "exponent": 18
+        }
+      ],
+      "base": "aepix",
+      "name": "Epix Testnet",
+      "display": "tepix",
```
Comment (JeremyParish69) [inline testnets/epixtestnet/assetlist.json:19]: this unit is not defined. Either use display 'epix' or update the denom_units

**Pull request merged**